tests/plugins.test.ts

```diff
@@ -64,10 +64,8 @@ test('setupPlugins and runProcessEvent', async () => {
     expect(pluginConfig.config).toEqual(pluginConfig39.config)
     expect(pluginConfig.error).toEqual(pluginConfig39.error)
 
-    expect(pluginConfig.plugin).toEqual({
-        ...plugin60,
-        capabilities: { jobs: [], scheduled_tasks: [], methods: ['processEvent'] },
-    })
+    expect(pluginConfig.plugin).toEqual(plugin60)
```

Comment:
this seems like a regression introduced in https://github.com/PostHog/plugin-server/runs/3059157937 where these tests didn't run